pkg/api/defaults_test.go

```diff
@@ -3175,7 +3175,7 @@ func TestDefaultEnablePodSecurityPolicy(t *testing.T) {
 					MasterProfile: &MasterProfile{},
 				},
 			},
-			expected: true,
+			expected: false,
```

Comment:
these UT changes reflect that `EnablePodSecurityPolicy` is now functionally deprecated